tests/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = {
+	env: {
+		mocha: true
```

Comment:
Mocha tests should be moved to `node-tests` folder as we do in other addons.

This file can be moved to root `.eslintrc`.

Reply:
Yeah, we plan to rename this folder when we include ember tests